exampleSite/config.toml

```diff
@@ -24,6 +24,8 @@ DefaultContentLanguage           = "en"                    # Default language fo
         localTime                = true                    # Show your current local time in contact section
         timeZone                 = "America/Los_Angeles"   # Your timezone as in the TZ* column of this list: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
         timeFormat               = "h:mm A"                # https://momentjs.com/docs/#/displaying/format/
+        timeRefresh              = 0                       # Update the localtime every timeRefresh milliseconds. This is usually either 0 (disabled) or 1000 (every second) 
```

Comment:
Why would someone configure this to something different the 1000?

Reply:
I can imagine that someone might want to reduce the load of the browser and put 2000 or even 5000 here. The time would be wrong by a couple of seconds, but this might be acceptable.

Reply:
Would you strongly be fine with removing this for now? I don't really see the necessity for a setting.

Reply:
No problem. Do you still want a boolean to enable/disable the refresh or shall I just enforce the refresh every second ?

Reply:
Let's just go with the simples solution and enforce a refresh.

Reply:
Done